server/bin/configure-apt.sh

```diff
@@ -10,4 +10,8 @@ cat > /etc/apt/preferences.d/preferences << EOL
 Package: *
 Pin: release a=unstable
 Pin-Priority: 450
+
+Package: *
+Pin: release a=trixie
```

Comment:
Maybe this should just be `testing`? Given the other one is just called `unstable`.

Reply:
Might need to add trixie next to sid on line 6? I'm not very familiar with how to set the package manager up on debian

Reply:
> Maybe this should just be `testing`? Given the other one is just called `unstable`.

We have the choice. Since `libio-compress-brotli-perl` is the only package from the trixie repo, it makes sense to use `trixie`. If you want to move all the packages from the unstable to the testing repo, it's probably better to use `testing` (maybe just add a comment to not forget to move `libio-compress-brotli-perl` from `testing` to `main` once `trixie` is released).

> Might also need to add trixie next to sid on line 6? I'm not very familiar with how to set the package manager up on debian

*Should* be fixed

Reply:
TBH Debian Testing isn't that different from using a non-LTS Ubuntu release. It's not that big of a deal if it just specifies testing.